Microsoft.Recognizers.Text.DateTime/Spanish/Extractors/SpanishDurationExtractorConfiguration.cs

```diff
@@ -8,8 +8,15 @@ public class SpanishDurationExtractorConfiguration : IDurationExtractorConfigura
         public static readonly Regex UnitRegex =
             new Regex(
                 @"(?<unit>años|año|meses|mes|semanas|semana|d[ií]as|d[ií]a|horas|hora|h|hr|hrs|hs|minutos|minuto|mins|min|segundos|segundo|segs|seg)\b",
-                RegexOptions.IgnoreCase | RegexOptions.Singleline);
-
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        //TODO: change to Spanish the AndRegex
+        public static readonly Regex SuffixAndRegex = new Regex(@"(?<suffix>\s*(and)\s+((an|a)\s+)?(?<suffix_num>half|quarter))",
+            RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        //TODO: modify the FollowedUnit to be like the follewing code
+        //       public static readonly Regex SuffixAndRegex = new Regex(@"(?<suffix>\s*(and)\s+((an|a)\s+)?(?<suffix_num>half|quarter))",
+        //       RegexOptions.IgnoreCase | RegexOptions.Singleline);

```

Comment:
I think this one can stay, as there is a TODO related to it.
However, it would be better if the TODO is implemented. Or at least an issue in github is created describing what needs to be done.